zeroize/Cargo.toml

```diff
@@ -27,9 +27,11 @@ default = ["alloc"]
 alloc = []
 std = ["alloc"]
 
-aarch64 = [] # NOTE: vestigial no-op feature; AArch64 support is always enabled now
+aarch64 = []                # NOTE: vestigial no-op feature; AArch64 support is always enabled now
 derive = ["zeroize_derive"]
-simd = [] # NOTE: MSRV 1.72
+simd = []                   # NOTE: MSRV 1.72
+
+test-allocator = ["std"]
```

Comment:
I don't think you need `std` for this.

Reply:
There shouldn't be any need for an additional crate feature at all, IMO

Reply:
`std` is needed because of the `use std::alloc::{GlobalAlloc, Layout, System};` – the custom allocator falls back to the system allocator.

Reply:
You don't need `std` for this. The same functionality is available in the `alloc` crate.

But considering that tests are built with `std` by default, it does not matter either way. You can simply remove the feature.

Reply:
…and the feature is there to stop `#[global_allocator]` from kicking in for all tests all the time.

Reply:
> You don't need `std` for this. The same functionality is available in the `alloc` crate.

You are correct, unless you need the system allocator, `System`. 

That's only available in `std::alloc` (compare alloc crate [here](https://doc.rust-lang.org/alloc/alloc/index.html#structs), std::alloc [here](https://doc.rust-lang.org/std/alloc/index.html#structs))